### PR TITLE
Separateinstall

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue #113.

### Summary of Changes
Insert the the method  public function runInstallAndTest into robofile.php

### Testing Instructions
Apply this changes and try the command 
tests/codeception/vendor/bin/robo run:install-and-test
Joomla will be installed and after that you can choose a test feature.

### Documentation Changes Required
The command 
tests/codeception/vendor/bin/robo run:install-and-test 
starts a new installation, disables Statistics, sets Error Reporting to Development and you can choose a test. I have updated Readme.md as well. Relevant after line 183.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +187,19 @@ If you want to see steps then you can use `--steps` option of codeception. Check
...
+#### You can run an install before individual run `feature` using following command.
+
+**_Linux & Mac_**
+```bash
+$ tests/codeception/vendor/bin/robo run:install-and-test
+```
+
+**_Windows_**
+```cmd
+$ tests\codeception\vendor\bin\robo run:install-and-test
```
Comment (yvesh) [inline README.md:199]: Hmm is the command not a bit long? :) How about we use run:test and you can add --install as parameter?

File: RoboFile.php [hunk 2]
```diff
@@ -389,27 +361,31 @@ public function runTests($opts = ['use-htaccess' => false, 'env' => 'desktop'])
...
 
-		$this->taskCodecept($pathToCodeception)
-			->arg('--steps')
-			->arg('--debug')
-			->arg('--fail-fast')
-			->arg('--env ' . $opts['env'])
-			->arg($this->testsPath . 'acceptance/frontend/')
-			->run()
-			->stopOnFail();
+                $this->loadFeatures($pathToTestFile,$suite);
```
Comment (yvesh) [inline RoboFile.php]: Space after comma

File: README.md (hunk 1 above)
Comment (astridx) [reply README.md:199]: I like it like it is. You do not need the whole command. If you type
`tests\codeception\vendor\bin\robo run:i`
the "installandtest" methode will be executed. 
But the long name explained exactly what the methode will do excetly.

File: RoboFile.php (hunk 2 above)
Comment (astridx) [reply RoboFile.php]: I have inserted a space :)
Next week I will reinstall my code styles ...

**Comment by astridx** [comment]
I do not like this an more and there are also overtaken changes.

**Pull request closed without merging**